File: C++/Class.cpp
#include<iostream>

using namespace std;

class Student
{
    public:
        int id;
        float gpa;

        void display()
        {
            cout<< id << " " <<  gpa <<endl;
        }

        void setValue(int x, float y)
        {
            id = x;
            gpa = y;
        }

};

int main()
{
    Student Rana, Anwar;

    /*Rana.id = 1929;
    Rana.gpa = 88.11;
    Anwar.id = 87;
    Anwar.gpa = 45.56;*/

    Rana.setValue(12, 48.3);
    Rana.display();
    Anwar.setValue(23, 88.7);
    Anwar.display();


    //cout<< Rana.id << endl;
    //cout<< Anwar.gpa << endl;
}

File: JavaScript/class.js
class Student {
    constructor(name, age) {
        this.name = name
        this.age = age
    }

    set sname(n) {
        this.name = n
    }
}

let s1 = new Student("Rana Ahamed", 104);
console.log(s1);

s1.sname = "Anwar"
console.log(s1);